kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java

```diff
@@ -124,6 +129,29 @@ protected void runGetTaskFormTest( long processInstanceId ) {
         }
     }
 
+    @Test(expected = KieServicesException.class)
+    public void testGetTaskFormWithoutPermissioneViaUIClientTest() throws Exception {
+        changeUser(USER_YODA);
+        long processInstanceId = processClient.startProcess(CONTAINER_ID, HIRING_PROCESS_ID);
+        assertTrue(processInstanceId > 0);
+
+        List<TaskSummary> tasks = taskClient.findTasksByStatusByProcessInstanceId(processInstanceId, null, 0, 10);
+        assertNotNull(tasks);
+        assertEquals(1, tasks.size());
+
+        Long taskId = tasks.get(0).getId();
+
+        changeUser(USER_JOHN);
+
+        assertClientException(
+                () -> uiServicesClient.getTaskForm(CONTAINER_ID, taskId, "en"),
+                401,
+                MessageFormat.format(TASK_PERMISSION_ERROR, taskId));
+
+        changeUser(USER_YODA);
+        processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
```

Comment:
@pefernan Please put 
`changeUser(USER_YODA);`
`processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);`
into the final statement of try clause covering assertClientException. That way the user is changed and process is aborted even in case the assert fails.

Reply:
@sutaakar done. Better now?